src/pages/JobOwner/Home/Home.tsx

```diff
@@ -64,11 +65,15 @@ export default function ClientHome() {
             {t("Dashboard.buttons.seeDrafts")}
           </Button>
         </S.TitleContainer>
-        {mockPosts.map(({ createdAt, name }) => (
-          <div key={name} style={{ margin: "15px 0", width: "100%" }}>
-            <JobPost createdAt={createdAt} name={name} />
-          </div>
-        ))}
+        {drafts ? (
+          drafts.map(({ createdAt, title }) => (
+            <div key={title} style={{ margin: "15px 0", width: "100%" }}>
```

Comment:
Same here, can the id be the key?

Also, below this code, the stepsList object is also missing the key property
```typescript
{stepsList.map((item) => (
  <li style={{ padding: "5px 0" }}>{item}</li>
))}
```